helm/aladdin-demo/templates/aladdin-demo.deploy.yaml

```diff
@@ -31,16 +31,21 @@ spec:
       - name: {{ .Chart.Name }}
         # Docker image for this container
         image: {{ .Values.deploy.ecr }}{{ .Chart.Name }}:{{ .Values.deploy.imageTag }}
-        workingDir: /home/{{ .Chart.Name}} 
+        workingDir: /home/{{ .Chart.Name }} 
         # Container port that is being exposed within the node
         ports:
         - containerPort: {{ .Values.app.port }}
           protocol: TCP
         # Mount the configuration file into the docker container
         volumeMounts:
           # Absolute path is used here
-          - mountPath: /config/
+          - mountPath: /home/{{ .Chart.Name }}/config/uwsgi.yaml
```

Comment:
if I change chart name, entrypoint.sh will fail. I recommend hard coding here

Reply:
Good catch, though I think it would be better to just remove the project name from the path everywhere rather than hard coding it here.

Reply:
helm is unable to replace source code (not a good idea either).
So I am not sure how you plan to use chart name in source.
Maybe there is an env variable or something.